src/main/java/hudson/remoting/Response.java

```diff
@@ -36,7 +37,7 @@
  * @see Request
  * @since 3.17
  */
-public final class Response<RSP,EXC extends Throwable> extends Command {
+public final class Response<RSP extends Serializable,EXC extends Throwable> extends Command {
```

Comment:
Instances of `Response` are always created from instances of `Request`, whose `RSP` type parameter is bounded by `Serializable`. So there is no reason not to similarly bound the type parameter of `Response`. Doing so makes the relationship between the two more clear. Note that the bound of the `EXC` type parameter on the `Response` class may be different (broader) than that of the `Request` class due to the fact that a response may include a `RequestAbortedException` as described above.